dev/tests/integration/testsuite/Magento/Security/Model/UserExpiration/ValidatorTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\Security\Model\UserExpiration;
+
+use Magento\Framework\Locale\ResolverInterface;
+use Magento\Framework\Stdlib\DateTime\TimezoneInterface;
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\User\Model\UserFactory;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Verify user expiration validation model.
+ */
+class ValidatorTest extends TestCase
+{
+    /**
+     * @var ResolverInterface
+     */
+    private $localeResolver;
+
+    /**
+     * @var TimezoneInterface
+     */
+    private $timeZone;
+
+    /**
+     * @inheridoc
+     */
+    protected function setUp(): void
+    {
+        $this->localeResolver = Bootstrap::getObjectManager()->get(ResolverInterface::class);
+        $this->timeZone = Bootstrap::getObjectManager()->get(TimezoneInterface::class);
+    }
+
+    /**
+     * Verify validation is passed on non US locale for date expiration.
+     *
+     * @magentoAppArea adminhtml
+     * @return void
+     */
+    public function testValidateUserExpiresAtWithNonUsLocale(): void
```

Comment:
It looks like it should be unit test.

Reply:
Test interacts with system configuration and there should be a lot of mock objects created to make it unit. I believe it's better to leave it as integration.